### ISSUE TITLE
Upgrading to 1.28.x or above causes previous data to disappear on UI.

### ISSUE THREAD

**Issue by vroad** [muety/wakapi#249]
**Describe the bug**
I have been running wakapi locally on docker running on mac. I have been using very old version of wakapi, which I don't know its exact version (version from Feb 2021). I tried upgrading it because I started getting errors on VSCode extension.

However, Upgrading wakapi to 1.28.x or above seemingly cause data loss. Maybe it's still on DB, I don't know the actual cause. The last version I confirmed to upgrade successfully is 1.27.3.

**System information**
* running on Docker Desktop for Mac
* Docker image tag: 1.28.3, 1.29.5
* Database: SQLite

**Comment by muety** [comment]
Hi @vroad! Can you describe a bit more detailed about what what data seems to be lost?

**Comment by vroad** [comment]
@muety I can still log in, so user passwords and api keys seem to be kept.

When I change the data range to "This year" in summary page, it shows the day I started using wakapi.
Graphs are not showing up at all. Number of top projects/operating systems/languages etc still change when I change date range. So it might just be the problem with the web interface.

I've also found errors in browser's console. I'm using Microsoft Edge 93.0.961.38. 

```
Uncaught TypeError: Cannot read properties of undefined (reading 'length')
    at app.js:23
    at Array.forEach (<anonymous>)
    at app.js:21
```

**Comment by Atakku** [comment]
@vroad @muety 
Having the same issue, using mariadb. Heartbeats are still in the DB, tried recalculating the summary but didn't help.
Using Mozilla Firefox 91.0.2, shows a similar error in browser's console:
```
Uncaught TypeError: Chart.defaults.global is undefined
    <anonymous> https://<domain>/assets/app.js:25
```
I checked the app.js file in question and it seemed to be outdated.
The culprit was Cloudflare's cache, and after forcefully purging it, the problem was resolved
